openapi/components/schemas/CustomField.yaml

```
@@ -41,10 +41,11 @@ properties:
       Custom fields of different types accept different constraints.
     properties:
       allowedValues:
-        description: List of allowed values. Available for custom fields with types `string` and `array`.
         oneOf:
           - type: string
+            description: List of allowed values. Available for custom fields with types `string`.
           - type: array
+            description: List of allowed values. Available for custom fields with types `array`.
```

nitpick: I think the 'type' of the field is something that defines it. Using 'with' sounds like 'type' is additional to the field.

```suggestion
            description: List of allowed values. Available for custom fields of type `string`.
          - type: array
            description: List of allowed values. Available for custom fields of type `array`.
```